testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-spark-hive-no-stats-fallback/hive.properties

```diff
@@ -0,0 +1,8 @@
+connector.name=hive
+hive.metastore.uri=thrift://hadoop-master:9083
+hive.metastore.thrift.use-spark-table-statistics=false
+hive.config.resources=/docker/presto-product-tests/conf/presto/etc/hive-default-fs-site.xml
+hive.allow-drop-table=true
+
+# Note: it's currently unclear why this one is needed, while also hive.orc.time-zone=UTC is not needed.
```

Comment:
FYI I'm checking whether this property is still needed
https://github.com/trinodb/trino/pull/16219